docs/Design/SoftArchitecture/MG.tex

```diff
@@ -187,13 +187,17 @@ \section{Module Hierarchy} \label{SecMH}
 actually be implemented.
 
 \begin{description}
-\item [\refstepcounter{mnum} \mthemnum \label{mHH}:] Hardware-Hiding Module
 \item [\refstepcounter{mnum} \mthemnum \label{mRVision}:] Receipt Vision Module (OCR)
+\item [\refstepcounter{mnum} \mthemnum \label{mReview}:] Receipt Extraction & Review
```

Comment:
I just called this one Receipt Extraction Module, the Review part will be still be part of it but just a submodule rather than in the entire Module name.